.typos.toml

```diff
@@ -25,6 +25,7 @@ extend-ignore-re = [
 "ede" = "ede"
 # Alternative spelling
 "unmarshaling" = "unmarshaling"
+"cancelations" = "cancelations"
```

Comment:
https://www.grammarpalette.com/cancellation-vs-cancelation-which-spelling/

> When in doubt, use “cancellation” with two “l’s.” It’s the preferred spelling in both American and British English, ensuring your writing remains clear and professional regardless of your audience.

Note that this one is different from the "canceled" / "cancelled" distinction, where there is a clear American vs British divide. There's some discussion on the page about why there's a difference, which touches briefly on why.

FWIW as an American English speaker, `cancelation` looks odd to me while `canceled` looks fine (and so does `cancelled` because I've worked with so many commonwealth English speakers :D).